### DISCUSSION TITLE
Wifi NIC for Management

### DISCUSSION THREAD

**Discussion by remrem29** [Security-Onion-Solutions/securityonion#2037] [General] [answered]
Hi Question about Wifi NICs

Is it possible to get a Wifi NIC to work with SO? I am aware that you have this line in the System Req. note

"Security Onion is designed to use wired interfaces. You may be able to make wireless interfaces work, but we don’t recommend or support it."

And I do understand that for sniffing part Wifi isn't the way to go, but what about the management interface?

As you have this line in the /home/secon/SecurityOnion/setup/so-common-functions file
"filtered_nics=$(ip link| awk -F: '$0 !~ "lo|vir|veth|br|docker|wl|^[^0-9]"{print $2}' | grep -vwe "$grep_string" | sed 's/"

under installation you will not be able to find your Wifi NIC as it would come up as "wlp0s26u1u6" or something like that and this is getting filtered out in the above  - I can of cause rename the NIC or remove the wl in the statement above but is there other ways for having a wifi nic as management interface?

**Comment by weslambert** [comment, marked as answer]
Not at this time -- we could consider supporting Wifi NICs for management in the future.
